common/src/main/kotlin/entity/Member.kt

```diff
@@ -18,8 +18,8 @@ data class DiscordGuildMember(
         val joinedAt: String,
         @SerialName("premium_since")
         val premiumSince: Optional<String?> = Optional.Missing(),
-        val deaf: Boolean,
-        val mute: Boolean,
+        val deaf: OptionalBoolean = OptionalBoolean.Missing,
+        val mute: OptionalBoolean = OptionalBoolean.Missing,
```

Comment:
You didn't change this in the data objects, does this compile?

Reply:
We don't expose the field anywhere in the Member object as of the time being. That's something we have to address